.clang-format

```diff
@@ -0,0 +1,49 @@
+---
```

Comment:
This is nothing to do with ocaml parser.
This one can be useful for all C files in ctags.
However, we have .indent.pro.